packages/babel-cli/test/index.js

```diff
@@ -13,6 +13,10 @@ const fs             = require("fs");
 const fixtureLoc = path.join(__dirname, "fixtures");
 const tmpLoc = path.join(__dirname, "tmp");
 
+const fileFilter = function(x) {
+  return x !== ".DS_Store";
```

Comment:
I guess this was making interference during the tests. Since .DS_Store are ignored in Git I think this shouldn't happen so often.